website/src/css/custom.css

```
@@ -1,695 +1,2181 @@
+  font-size: 0.85em;
+  font-variant-ligatures: none;
+}
+
+.markdown :not(pre) > code {
+  background-color: var(--et-inline-code-bg);
+  color: var(--et-inline-code-color);
+  border: 1px solid transparent;
+  border-radius: var(--et-radius-sm);
+  padding: 0.12em 0.36em;
+  font-weight: 500;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)